<launch>
    <node pkg="stereo_image_folder_publisher" type="image_folder_publisher.py" name="stereo_image_folder_publisher" output="screen">

        <param name="topic_name_left" value="image_raw_left" />
        <param name="topic_name_right" value="image_raw_right" />

        <param name="publish_rate" value="30"/>
        <param name="sort_files" value="True" />
        <param name="frame_id_left" value="camera_left" />
        <param name="frame_id_right" value="camera_right" />
        <param name="sleep" value="0" />
        <param name="loop" value="-1" />
        <param name="image_folder_right" value="/home/avanna/dataset/left" />
        <param name="image_folder_left" value="/home/avanna/dataset/right" />

    </node>
</launch>